### PR TITLE
Admin Config Rank

### PR DESCRIPTION
**Pull request by SpaceLoveSs13**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl:
add: Added Babymin rank
add: Babymin has more permissions now

/:cl:

## About The Pull Request
 
Gives more fun and more space for experienced admins who have been told to stay as "Babymin"

## Why It's Good For The Game

This is to add extra permissions to a newly added rank Babymin as it was asked by Gamemasters and Headmins. 

I was told to make changes by myself because I am supposed to stay Babymin always. Took this is as an opportunity to make my first PR. If I have fucked up anything please say.

### CODE REVIEW

**Comment by SpaceLoveSs13** [comment]
Can someone help me figure out the travis-ci check by the way? Although if it gets merged without the check and doesn't conflict, that will be fine too.

File: config/admin_ranks.txt [hunk 1]
```diff
@@ -83,3 +83,8 @@ Name = Coder
 Include = DEBUG VAREDIT SERVER SPAWN POLL
 Exclude = AUTOADMIN
 Edit =
+
+Name = Babymin
+Include = EVERYTHING
```
Comment (AnthonyJNgo) [inline config/admin_ranks.txt:88]: This would give you every permission in the game.

File: config/admins.txt [hunk 2]
```diff
@@ -135,3 +135,4 @@ Naksuasdf = Game Master
 MrDoomBringer = Game Master
 shizcalev = Game Master
 NicBR = Game Master
+SpaceLove = Babymin
```
Comment (AnthonyJNgo) [inline config/admins.txt:138]: This doesn't need to be a PR we can do this on the server using server tools.

**Comment by AnthonyJNgo** [comment]
You made a PR successfully, however that PR is being rejected for being ridiculous.

**Pull request closed without merging**